test/e2e/connectivity_test.go

```diff
@@ -250,10 +250,10 @@ func testPodConnectivityDifferentNodes(t *testing.T, data *TestData) {
 	data.testPodConnectivityDifferentNodes(t)
 }
 
-func (data *TestData) redeployAntrea(t *testing.T, option deployAntreaOptions) {
+func (data *TestData) redeployAntrea(t *testing.T, option deployAntreaConfiguration) {
 	var err error
 	// export logs before deleting Antrea
-	exportLogs(t, data, fmt.Sprintf("beforeRedeploy%s", option), false)
+	exportLogs(t, data, fmt.Sprintf("beforeRedeploy %s", option.name), false)
```

Comment:
I think spaces in file names are not ideal, could we keep it the same or use a dash (`-`) separator?

Reply:
Changed back.